projectile.el

```diff
@@ -530,6 +530,22 @@ See also `projectile-update-mode-line'."
   :type 'function
   :package-version '(projectile . "2.0.0"))
 
+(defcustom projectile-default-src-directory "src/"
+  "The default value of a project's src-dir property.
```

Comment:
Sorry I'm not sure the convention for `:package-version`, do you wait until a new release before adding this?